client/src/ehr_components/NewPatient.tsx

```diff
@@ -0,0 +1,245 @@
+import { useHistory } from "react-router-dom";
+import { toastr } from "react-redux-toastr";
+import React, { useState } from "react";
+import Axios from "axios";
+
+import FieldRenderer from "./common_components/field-renderer";
+import { stringify } from "querystring";
+

```

Comment:
I'm not sure why you added stringify querystring, doesn't look like you use it anyway

Reply:
I'm not sure why that's there either. Regardless, I removed it in recent commits!